apriltag_ros/msg/AprilTagDetection.msg

```diff
@@ -11,4 +11,10 @@ float64[] size
 # tag, the homography is from the four tag corners. If a tag bundle, the
 # homography is from at least the four corners of one member tag and at most the
 # four corners of all member tags.
-geometry_msgs/PoseWithCovarianceStamped pose
\ No newline at end of file
+geometry_msgs/PoseWithCovarianceStamped pose
+int32[2] pix_tl
+int32[2] pix_tr
+int32[2] pix_br
+int32[2] pix_bl
```

Comment:
Since the underlying data is floating point, the message should not be int.

Also, everywhere internally, these corners are stored as 4-long array. You should probably do the same here.

In the interest of reusing datatypes, I'd suggest the following:
`geometry_msgs/Point32[4] pixel_corners`